SearchHistory/migrations/0002_test_data.py

```diff
@@ -0,0 +1,25 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('SearchHistory', '0001_initial'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        from SearchHistory.models import SearchHistory
+
+        test_data = [
+            ('1', 'Test Pro1', 'Test Client1'),
+            ('2', 'Test Pro2', 'Test Client2'),
+            ('3', 'Test Pro3', 'Test Client3'),
+            ('4', 'Test Pro4', 'Test Client4'),
+        ]
+
+        with transaction.atomic():
```

Comment:
What is the reason for using an `atomic` transaction here?

Reply:
@dkeler 
https://docs.djangoproject.com/en/4.2/howto/writing-migrations/#non-atomic-migrations